readme.md

```diff
@@ -58,7 +58,12 @@ The aim is to completely decompile RollerCoaster Tycoon 2 into C so that cross-p
 
 # 2 Downloading the game / Building the source code
 
-A third-party offers [downloadable precompiled builds](https://openrct2.com/download). However, building the project is always recommended.
+A couple of third parties offer downloadable precompiled builds. However, building the project is always recommended
+
+[OpenRCT2.com](https://openrct2.com/download)  
+[OpenRCT.net](https://openrct.net/builds.php)
+
+There is also a Launcher available from [OpenRCT.net](https://openrct.net/download) that will automatically update your build so that you always have the current version as they are released.
```

Comment:
It is a noun, which is not capitalised in English, except in names or titles. This is neither of those two. (Persons and Places are capitalised because they are names.)